courses/fundamentals_of_ada/adv_240_tasking.rst

```diff
@@ -372,6 +427,75 @@ Example: Protected Objects - Body
 Quiz
 ------
 
+.. code:: Ada
+
+    procedure Main is
+        protected type O is
+           entry P;
+        end O;
+
+        protected body O is
+        begin
+           entry P is
+              Put_Line ("OK");
```

Comment:
probably should have with/use of Ada.Text_io

Reply:
I disagree, this overloads the slides for a piece of info that is repeated 100 times over in the content. Especially since this is an advanced course.